### ISSUE TITLE
[Sharing] Shortcuts 

### ISSUE THREAD

**Issue by clshi2022** [markusfisch/PieLauncher#111]
This is not an issue, so feel free to close this anytime.

Just sharing for awareness, since i dun see it written anywhere:
In the App drawer search,

- hitting "," twice opens the Pie Menu editor
- hitting "." twice opens the settings menu

I've found these shortcuts useful, especially the one to open the Pie Menu Editor

**Comment by markusfisch** [comment]
Indeed, I should mention this somewhere. Keep it open as a reminder to do so 😉

**Comment by jsnfwlr** [comment]
![photostudio_1722886252796.png](https://github.com/user-attachments/assets/a451bf21-2e86-444d-8c61-18603a32c889)

Might be worth adding a button to the left of the search input (balancing the pencil) with a into icon that opens a dialog/page with documentation for these kind of features...
